db/migrate/20160324174218_create_container_deployment_nodes.rb

```diff
@@ -0,0 +1,16 @@
+class CreateContainerDeploymentNodes < ActiveRecord::Migration[5.0]
+  def change
+    create_table :container_deployment_nodes do |t|
+      t.string :ip_or_hostname
+      t.string :name
+      t.text :labels
+      t.references :container_deployment
+      t.references :vm
+      t.text :docker_storage_devs, :array => true, :default => []
+      t.string :docker_storage_data_size
+      t.boolean :deploy_metrics, :default => true
+      t.boolean :deploy_registry, :default => true
```

Comment:
> docker_storage_device in CreateContainerDeploymentNodes
>  deploy_metrics and deploy_registry boolean in ContainerDeployment, double-check the UI to see what were the other options we had there (in case we missed them)

@simon3z Added those fields over here.